.manage.py

```diff
@@ -0,0 +1,47 @@
+from argparse import ArgumentParser
+from contextlib import chdir
+from pathlib import Path
+from shutil import rmtree, move
+from subprocess import call
+from tempfile import TemporaryDirectory
+
+VERSION = '3.12'
+
+
+def update_pots() -> None:
+    """updates translation sources"""
+    _call('git diff --exit-code')
+    with TemporaryDirectory() as directory:
+        with chdir(directory):
+            _clone_cpython_repo(VERSION)
+            _build_gettext()
+        rmtree('.pot')
+        move(Path(directory) / 'cpython/Doc/locales/pot', '.pot')
+    if (call("git diff -I'^\"POT-Creation-Date: ' --exit-code", shell=True)) != 0:
+        _call('git add .')
+        _call('git commit -m "Update sources"')
+
+
+def _clone_cpython_repo(version: str):
+    _call(f'git clone -b {version} --single-branch https://github.com/python/cpython.git --depth 1')
+
+
+def _build_gettext():
+    _call(
+        "make -C cpython/Doc/ ALLSPHINXOPTS='-E -b gettext -D gettext_compact=0 -d build/.doctrees . locales/pot' build"
```

Comment:
`make venv` is required as well to install dependencies, isn't it?

If you want to rely on other virtual environment, I believe you can drop 'make' and use sphinx-build -b gettext directly (with probably the same arguments).

Reply:
I believe the command inside falls back to external environment, that's why it doesn't fail now.

I'm leaning towards adding `make venv` instead, as I need to setup an env for action workflow and for now the script doesn't require Sphinx.

Good catch, thanks!